data-msgpack.cabal

```diff
@@ -1,5 +1,5 @@
 name:                 data-msgpack
-version:              0.0.6
+version:              0.0.8
```

Comment:
Version bump shouldn't be there in the first place. It should be in it's own commit.